apps/test/unit/code-studio/components/progress/UnitOverviewTopRowTest.jsx

```diff
@@ -131,23 +129,6 @@ describe('UnitOverviewTopRow', () => {
     ).toBe(true);
   });
 
-  it('renders SectionAssigner for instructor', () => {
```

Comment:
Removing tests that check for removed elements. Will add unit tests for UnitOverviewActionRow as a follow up